src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -554,6 +554,11 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('explorer.incrementalNaming', "Controls which naming strategy to use when giving a new name to a duplicated Explorer item on paste."),
 			default: 'simple'
 		},
+		'explorer.autoOpenDroppedFile': {
+			'type': 'boolean',
+			'description': nls.localize('autoOpenDroppedFile', "Controls whether the Explorer should automatically open a file when it is dropped into the explorer"),
+			'default': true
```

Comment:
@lramos15 not sure this should be the default?

Reply:
It is the default right now @Tyriar.

When you drop a file in it automatically opens. That's the complaint https://github.com/microsoft/vscode/issues/168809